packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -4,12 +4,14 @@ import * as React from 'react';
 import {
   BaseComponent,
   autobind,
+  css,
   divProperties,
   getNativeProps
 } from '../../Utilities';
 import { ITooltipHostProps } from './TooltipHost.Props';
 import { Tooltip } from './Tooltip';
 import { TooltipDelay } from './Tooltip.Props';
+import styles from './Tooltip.scss';
```

Comment:
If TooltipHost is a separate component, it should have its own stylesheet right?